wal.go

```diff
@@ -389,9 +393,10 @@ func (w *SegmentWAL) Truncate(mint int64, keep func(uint64) bool) error {
 	csf.Close()
 
 	candidates[0].Close() // need close before remove on platform windows
-	if err := renameFile(csf.Name(), candidates[0].Name()); err != nil {
+	if err := fileutil.Rename(csf.Name(), candidates[0].Name()); err != nil {
```

Comment:
this one looks like should be `fileutil.Replace` as `candidates[0].Name()` might already exist.

Reply:
fixed